frontend/package.json

```diff
@@ -32,11 +32,18 @@
     "expo-constants": "^7.0.0",
     "expo-font": "^7.0.0",
     "expo-web-browser": "^7.0.0",
+    "google-map-react": "^1.1.5",
+    "install": "^0.13.0",
```

Comment:
I am curious what 'install' is 😮

Reply:
i still don't know but i uh believe in u

Reply:
Ah I have realized I need to remove `google-map-react` from this commit. Will do in master